index.html

```diff
@@ -21,6 +21,12 @@
         <button id="run">Run!</button>
         <button id="reset">Reset!</button>
         <br/>
+        <input type = "file" id = "choose_file" accept=".txt,.py">
+        <button id = "load">Load</button>
+        <br/>
+        <label for = "save_title">title:</label>
```

Comment:
Maybe we can name this `title` to something a bit more descriptive, like "File name you want to save"?

Reply:
Sure, I'll modify it.